CHANGELOG.rst

```diff
@@ -1,5 +1,26 @@
 Changelog
 =========
+1.19.0 (2021-07-14)
+-------------------
+MLflow 1.19.0 includes several major features and improvements:
+
+Features:
+
+- Add support for plotting per-class feature importance computed on linear boosters in XGBoost autologging (#4523, @dbczumar)
+- Add ``mlflow_create_registered_model`` and ``mlflow_delete_registered_model`` for R to create/delete registered models.
+- Add support for setting tags while resuming a run (#4497, @dbczumar)
+- MLflow UI updates (#4490, @sunishsheth2009)
+
+  - Add i18n support.
```

Comment:
Does this sound like we're already supporting several languages?

Reply:
We can change this to say "Add framework for internationalization support"

Reply:
Thanks, done!